pkg/db/db.go

```diff
@@ -7,7 +7,7 @@ import (
 	"log"
 	"regexp"
 
-	"github.com/ExpansiveWorlds/instrumentedsql"
+	"github.com/luna-duclos/instrumentedsql"
```

Comment:
Is this intended to be part of this change?

Reply:
yeah, it was a fork I found to be more up-to-date since the ExpansiveWorlds project hasn't been touched since 2017 and won't compile anymore.